tests/rst_output_pytest.py

```diff
@@ -20,7 +20,9 @@
     "\n",
     ":gherkin-feature-keyword:`Feature:` :gherkin-feature-content:`Testing Sphinx Writer`\n",
     "====================================================================================\n\n",
-    "    :gherkin-feature-description:`Test the ability to write Gherkin data to a SphinxWriter object`\n",
+    "\n",
+    "    :gherkin-feature-description:`    Test the ability to write Gherkin data to a SphinxWriter object`\n",
```

Comment:
Why do we see all the `gherkin-feature-description` lines with extra space before the text in this diff (in non-raw outputs)? I'm trying to figure out where that's coming from and I'm not seeing it.

Reply:
Yeah that's a misexplanation on my part. We updated the commit on pytest-bdd dependency, so we get this new change I made: https://github.com/rbcasperson/pytest-bdd/commit/b3d464bef1af2accbef493c6be85fc6232c318ed

That gets rid of line stripping for descriptions. Without that we could have no hope of getting raw rST because the indentation would not be preserved. So this is a byproduct of that pytest-bdd change. What we could do is put the stripping back into the sphinx-gherkindoc logic when this new flag is _not_ used, but I'm not seeing it having an effect.

Reply:
Ah, that makes sense. So long as it doesn't impact the generated HTML, seems fine to me.

Reply:
Yeah, I want to verify this by doing a build on my work repo tomorrow...